### ISSUE TITLE
EQL correctness tests failing on 7.9

### ISSUE THREAD

**Issue by DaveCTurner** [elastic/elasticsearch#63927]
**Build scan**:

https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+7.9+eql-correctness-tests/12/console

**Repro line**:

Not given.

**Reproduces locally?**:

Not sure how to try. Looks to be CI-environment-related?

**Applicable branches**:

7.9

**Failure history**:

[Build stats](https://build-stats.elastic.co/app/kibana#/discover?_g=(refreshInterval:(pause:!t,value:0),time:(from:now-90d,mode:quick,to:now))&_a=(columns:!(build.branch),filters:!(('$state':(store:appState),meta:(alias:!n,disabled:!f,index:b646ed00-7efc-11e8-bf69-63c8ef516157,key:build.job-name-extra,negate:!f,params:(query:eql-correctness-tests,type:phrase),type:phrase,value:eql-correctness-tests),query:(match:(build.job-name-extra:(query:eql-correctness-tests,type:phrase)))),('$state':(store:appState),meta:(alias:!n,disabled:!f,index:b646ed00-7efc-11e8-bf69-63c8ef516157,key:process.status,negate:!f,params:(query:FAILURE,type:phrase),type:phrase,value:FAILURE),query:(match:(process.status:(query:FAILURE,type:phrase))))),index:b646ed00-7efc-11e8-bf69-63c8ef516157,interval:auto,query:(language:lucene,query:''),sort:!(process.time-start,desc))) indicate this has been consistently failing on 7.9 since inception, but works ok on the other branches.

**Failure excerpt**:

```
10:05:20 runbld>>> >>>>>>>>>>>> SCRIPT EXECUTION BEGIN >>>>>>>>>>>>
10:05:20 runbld>>> Adding /var/lib/jenkins/.java/openjdk-14.0.2+12-linux/bin to the path.
10:05:20 + [[ refs/heads/7.9 == \r\e\f\s\/\h\e\a\d\s\/\6\.\8 ]]
10:05:20 + set -euo pipefail
10:05:20 + rm -Rfv /var/lib/jenkins/.gradle/init.d
10:05:20 + mkdir -p /var/lib/jenkins/.gradle/init.d
10:05:20 + cp -v /dev/shm/elastic+elasticsearch+7.9+eql-correctness-tests/.ci/init.gradle /var/lib/jenkins/.gradle/init.d
10:05:20 '/dev/shm/elastic+elasticsearch+7.9+eql-correctness-tests/.ci/init.gradle' -> '/var/lib/jenkins/.gradle/init.d/init.gradle'
10:05:20 + set +x
10:05:20 /tmp/jenkins8551424652428397105.sh: line 15: /dev/shm/elastic+elasticsearch+7.9+eql-correctness-tests/x-pack/plugin/eql/qa/correctness/credentials.gcs.json: No such file or directory
10:05:20 runbld>>> <<<<<<<<<<<< SCRIPT EXECUTION END <<<<<<<<<<<<
```

**Comment by matriv** [comment]
@mark-vieira Could you please check that?
As discussed is shouldn't be enabled for 7.9, but from 7.10 onwards (currently 7.10, 7.x and master).

**Comment by mark-vieira** [comment]
I have a PR open to disable this on the 7.9 branch.

https://github.com/elastic/infra/pull/24233/files#diff-301b6b3f9d09d3ef2dfc20617f8d277ee6de57a2c2ff67dc8ec9dde1ff3916d7R230

**Comment by mark-vieira** [comment]
That PR has been merged so this job is now a no-op on 7.9.
